listener.py

```
@@ -135,9 +316,39 @@ def process_pending(cfg: dict):
             except Exception:
                 pass
 
+    # ── Mark-read jobs (single browser session for all groups) ────────────────
+    if mark_read_jobs:
+        group_names = [j["name"] for j in mark_read_jobs]
+        for job in mark_read_jobs:
+            try:
+                api_post(f"{railway_url}/api/scrape-queue/{job['id']}/update",
+                         api_key, {"status": "running"})
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)